pkg/api/converterfromapi.go

```diff
@@ -998,6 +998,7 @@ func convertAgentPoolProfileToVLabs(api *AgentPoolProfile, p *vlabs.AgentPoolPro
 	p.Name = api.Name
 	p.Count = api.Count
 	p.VMSize = api.VMSize
+	p.CustomVMTags = api.CustomVMTags
```

Comment:
You're missing the `convertMasterProfileToVLabs` line

Reply:
You are right! Just did this too. Thanks!